.github/workflows/ci.yml

```
@@ -376,7 +376,7 @@ jobs:
         run: ./test/Scripts.Integration.Test/build-project.ps1 -UnityPath "${{ env.UNITY_PATH }}"
```

```suggestion
        run: ./test/Scripts.Integration.Test/build-project.ps1 -UnityPath "${env:UNITY_PATH}"
```
I think this one also needs to be changed.